lib/iris/cube.py

```diff
@@ -3432,7 +3431,7 @@ def regrid(self, grid, scheme):
         return regridder(self)
 
 
-class ClassDict(object, UserDict.DictMixin):
+class ClassDict(collections.MutableMapping, object):
```

Comment:
To be clear, `MutableMapping` does _not_ inherit from `object`, so it does make some functional difference.